chromium_src/components/search_engines/util.cc

```diff
@@ -6,10 +6,13 @@
 #define GetSearchProvidersUsingKeywordResult \
   GetSearchProvidersUsingKeywordResult_ChromiumImpl
 #include "src/components/search_engines/util.cc"
+
+#include <algorithm>
```

Comment:
Looks like a duplicate include with the one below.

Reply:
Fixed